Controller/ProfilerController.php

```diff
@@ -128,17 +125,6 @@ private function explainSQLServerPlatform(Connection $connection, array $query):
 
         $stmt = $connection->executeQuery($sql, $params, $query['types']);
 
-        // DBAL 2.13 "forward compatibility" BC break handling
-        if ($stmt instanceof Result) {
-            $stmt = $stmt->getIterator();
-        }
-
-        if (! $stmt instanceof PDOStatement) {
-            throw new LogicException('We need nextRowSet() functionality feature, which is not available with current DBAL driver');
-        }
-
-        $stmt->nextRowset();
-
         return $stmt->fetchAll(PDO::FETCH_ASSOC);
```

Comment:
This doesn't work with DBAL 3 either, does it. I'm tempted to replace this method with an exception and a call for contributors, tbh. 🙈

Reply:
yeah I have no easy way of testing this :cry: :see_no_evil: seems completely broken indeed

Reply:
Although maybe it works?

That method should still work: https://github.com/doctrine/dbal/blob/3.5.x/src/Result.php#L300

Reply:
Yes, but it does not accept PDO constants anymore.

Reply:
Well... `FetchMode::ASSOCIATIVE = PDO::FETCH_ASSOC = 2` :see_no_evil:

Reply:
What a lucky coincidence. 😁 

Shall we just switch to `fetchAllAssociative()` and cross fingers then?